src/components/book-card/z-book-card/index.tsx

```diff
@@ -77,254 +66,165 @@ export class ZBookCard {
-                  </div>
-                  <div class="tags">{this.renderTags()}</div>
-                </div>
-                <div class="link-section">
-                  <div class="catalog-link">
-                    {this.catalogUrl && (
-                      <a
-                        href={this.catalogUrl}
-                        target="_blank"
-                        class="z-link z-link-blue z-link-icon body-4-sb"
-                        onClick={() => this.emitCatalogClick()}
-                      >
-                        Scheda catalogo
-                        <z-icon
-                          name="arrow-quad-north-east"
-                          height={16}
-                          width={16}
-                          fill="color-primary01-icon"
-                        ></z-icon>
-                      </a>
-                    )}
-                  </div>
-                  {this.adoption && <div class="adoption-tag body-5-sb">adottato</div>}
-                </div>
+                {this.renderVolumeTitle()}
+                {this.renderIsbn()}
               </div>
+              <slot name="tags"></slot>
+              <slot name="data"></slot>
             </div>
-            <div class="ebook-container">
-              <slot name="ebook">
-                {this.ebookUrl && (
-                  <div class="ebook">
-                    <div>
-                      <div class="app-name">
-                        <img
-                          class="ebook-logo"
-                          aria-hidden="true"
-                        />
-                        <div class="body-4-sb">
-                          <span class="laz">laZ</span> Ebook
-                        </div>
-                      </div>
-                      {this.variant === BookCardVariant.LANDSCAPE && this.reflowable && (
-                        <div class="body-5">
-                          Anche nella versione libro liquido con{" "}
-                          <button
-                            class="body-5-sb immersive-reader z-link"
-                            onClick={() => this.emitImmersiveReaderClick()}
-                            tabIndex={0}
-                          >
-                            strumento di lettura immersiva
-                          </button>
-                        </div>
-                      )}
-                    </div>
-                    <z-button
-                      size={ControlSize.X_SMALL}
-                      href={this.ebookUrl}
-                      target="_blank"
-                      onClick={() => this.emitEbookClick()}
-                      role="link"
-                      aria-description={`leggi l'ebook ${this.operaTitle} su laZ Ebook`}
-                    >
-                      leggi ebook
-                    </z-button>
-                  </div>
-                )}
-              </slot>
+            <div class="bottom">
```

Comment:
meglio fare un render di questo div in modo condizionale in base alla presenza di `ebookUrl` in modo da non averlo nel DOM vuoto ed evitare che il `gap` del div `.card-info` crei uno spazio vuoto in fondo alla card portrait:
<img width="329" alt="image" src="https://github.com/user-attachments/assets/9a77062d-8e70-4060-b2c1-88087024d1a8" />